rule/dubbo-2.6/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>io.opentracing.contrib.specialagent.rule</groupId>
     <artifactId>rule</artifactId>
-    <version>1.6.1-SNAPSHOT</version>
+    <version>1.6.1</version>
```

Comment:
Hi @limfriend, please rebase with upstream/master. The version is now `1.6.2-SNAPSHOT`.